PropertyChanged.Fody/CecilExtensions.cs

```diff
@@ -11,6 +11,26 @@ public static string GetName(this PropertyDefinition propertyDefinition)
         return $"{propertyDefinition.DeclaringType.FullName}.{propertyDefinition.Name}";
     }
 
+    public static bool IsCallToMethod(this Instruction instruction, MethodDefinition method)
+    {
+        if (!instruction.OpCode.IsCall())
+        {
+            return false;
+        }
+
+        if (!(instruction.Operand is MethodReference methodReference))
+        {
+            return false;
+        }
+
+        if (methodReference.Resolve() != method)
```

Comment:
Adding `if (methodReference.Name != method.Name) return false;` before the call to `Resolve` could be a nice optimization.

Reply:
:+1: